common/actions/transactions/constants.ts

```diff
@@ -1,5 +1,7 @@
 export enum TypeKeys {
   TRANSACTIONS_FETCH_TRANSACTION_DATA = 'TRANSACTIONS_FETCH_TRANSACTION_DATA',
   TRANSACTIONS_SET_TRANSACTION_DATA = 'TRANSACTIONS_SET_TRANSACTION_DATA',
-  TRANSACTIONS_SET_TRANSACTION_ERROR = 'TRANSACTIONS_SET_TRANSACTION_ERROR'
+  TRANSACTIONS_SET_TRANSACTION_ERROR = 'TRANSACTIONS_SET_TRANSACTION_ERROR',
```

Comment:
The above two action constants arent in this PR but, shouldn't this be the other way around like `TRANSACTION_TRANSACTION_DATA_SET`?

Reply:
Do we have a history of doing NAMESPACE_NOUN_VERB? Most places I see have the noun last. Either way, we should probably document nomenclature if we have a preference.